From dc81c748f528cf9bb68c9ef1cd016b4091e8c543 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 25 Jul 2018 10:20:08 +0800
Subject: [PATCH 41/65] live install supports kickstart

While live install, WRLinux does not support harddrive,
use cdrom to replace

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Update patch context for anaconda 32.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/installation.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pyanaconda/installation.py b/pyanaconda/installation.py
index 0f115655bb..1f420528a9 100644
--- a/pyanaconda/installation.py
+++ b/pyanaconda/installation.py
@@ -65,6 +65,10 @@ class WriteResolvConfTask(Task):
 def _writeKS(ksdata):
     path = conf.target.system_root + "/root/anaconda-ks.cfg"
 
+    # Correct method of live install, use cdrom to replace
+    if ksdata.method.method == "harddrive":
+        ksdata.method.method = "cdrom"
+
     # Make it so only root can read - could have passwords
     with util.open_with_perm(path, "w", 0o600) as f:
         f.write(str(ksdata))
-- 
2.7.4

